frontend/src/views/admin/assignments/manipulate-assignment-confirmation.tsx

```diff
@@ -39,33 +40,24 @@ const withdrawModalColumn = [
     },
 ];
 
-function compareString(str1: string, str2: string) {
-    if (str1 > str2) {
-        return 1;
-    } else if (str1 < str2) {
-        return -1;
-    }
-    return 0;
-}
-
-function compareAssignment(a1: Assignment, a2: Assignment) {
-    return (
-        compareString(a1.position.position_code, a2.position.position_code) ||
-        compareString(
-            a1.applicant.last_name || "",
-            a2.applicant.last_name || ""
-        ) ||
-        compareString(a1.applicant.first_name, a2.applicant.first_name)
-    );
-}
-
-export function MultiWithdrawOfferConfirmation(props: {
+export function MultiManipulateOfferConfirmation(props: {
     data: Assignment[];
     visible: boolean;
     setVisible: (visibility: boolean) => void;
-    withdrawOffers: () => void;
+    manipulateOffers: () => void;
+    titleMsg: String;
+    alertMsg: String;
```

Comment:
What is an `alertMsg`? The name doesn't tell me...

Avoid abbreviating. E.g., use `alertMessage` instead. (but don't actually, because `alertMessage` is not a good name for this variable.)

Reply:
After reading the code, it looks like `alertMsg` is actually `body`.